djangocms_blog/models.py

```diff
@@ -228,6 +228,7 @@ class Post(KnockerModel, BlogMetaMixin, TranslatableModel):
         title=models.CharField(_('title'), max_length=767),
         slug=AutoSlugField(_('slug'), max_length=767, blank=True,
                            db_index=True, allow_unicode=True),
+        subtitle=models.TextField(verbose_name=_('subtitle'), blank=True, default=''),
```

Comment:
I think we should use a charfield here

Reply:
Sure! Any idea about the max_length? Was thinking at 767 as the title, but maybe we could need more.

Reply:
767 is a maximum due to mysql fields constraint

Reply:
:+1: